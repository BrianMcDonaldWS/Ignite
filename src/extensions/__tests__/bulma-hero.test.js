import MarkdownIt from 'markdown-it';
import hero from '../bulma-hero';

const md = new MarkdownIt().use(hero);

test('bulma - hero', () => {
  expect(
    md.render(`
::: hero

# Title

## I can contain various combinations of elements.
:::
  `)
  ).toMatchSnapshot();
});
