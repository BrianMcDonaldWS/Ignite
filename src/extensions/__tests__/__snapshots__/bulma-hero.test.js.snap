// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bulma - hero 1`] = `
"
        <section class=\\"hero \\">
          <div class=\\"hero-body\\">
      <h1 class=\\"title\\">Title</h1>
<h2 class=\\"subtitle\\">I can contain various combinations of elements.</h2>

        </div>
      </section>
    "
`;

exports[`bulma - hero 2`] = `
"
        <section class=\\"hero \\">
          <div class=\\"hero-body\\">
      <h1 id=\\"title\\" class=\\"title\\">Title</h1>
<h2 id=\\"i-can-contain-various-combinations-of-elements.\\" class=\\"subtitle\\">I can contain various combinations of elements.</h2>

        </div>
      </section>
    "
`;
